### PR TITLE
[MNG-7474] SessionScoped beans should be singletons for a given session

### PR DESCRIPTION
**Pull request by gnodet**

### CODE REVIEW

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7474SessionScopeTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        throws Exception
+    {
+        File testDir = ResourceExtractor.simpleExtractResources( getClass(), "/mng-7474-session-scope" );
+        projectDir = new File( testDir, "project" );
+    }
+
+    public void testSessionScope()
+            throws Exception
+    {
+        File testDir = ResourceExtractor.simpleExtractResources( getClass(), "/mng-7474-session-scope" );
```
Comment (michael-o) [inline core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7474SessionScopeTest.java:54]: Why another extract?

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7474SessionScopeTest.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+        Verifier verifier = newVerifier( testDir.getAbsolutePath() );
+        verifier.executeGoal( "validate" );
+        verifier.verifyErrorFreeLog();
+        verifier.resetStreams();
+
+        List<String> logLines = verifier.loadFile( verifier.getBasedir(), verifier.getLogFileName(), false );
```
Comment (michael-o) [inline core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7474SessionScopeTest.java:61]: Why do you load them w/o any verification?

**Comment by michael-o** [comment]
Running ITs...

**Comment by michael-o** [comment]
After version bump of MRESOURCES all tests pass for me on Windows 10 and FreeBSD 12. Waiting for comment answers.

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7474SessionScopeTest.java (hunk 2 above)
Comment (gnodet) [reply core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7474SessionScopeTest.java:61]: It was for debugging iirc. I've removed it.

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7474SessionScopeTest.java (hunk 1 above)
Comment (gnodet) [reply core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7474SessionScopeTest.java:54]: Right, unneeded.

**Pull request merged**